src/index.html

```diff
@@ -1,12 +1,492 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page__body">
+    <header class="header page__header">
+      <a href="#home" class="header__logo-link">
+        <img
+         src="/images/logo.png"
+         alt="logo"
+         class="header__logo-img"
+        >
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+             href="#apple"
+             class="nav__link is-active"
+            >
+             Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#samsung"
+             class="nav__link"
+            >
+             Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#smartphones"
+             class="nav__link"
+            >
+             Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             href="#laptops & Computers"
```

Comment:
Use the '%20' character instead of the space in the href attribute for the 'Laptops & Computers' link: href="#laptops%20&%20computers"